rosidl_generator_rs/msg/Bool.msg

```diff
@@ -1 +0,0 @@
-bool empty_bool
```

Comment:
Why remove the test messages for the generator?

Reply:
I thought they weren't used. Can we depend on the test_msgs package instead ?

Reply:
The `test_msgs` package depends on the generator to be installed before getting there (i.e. you can't have `test_msgs` built unless you have built the generators). However when you're testing the generator itself, you need to have the messages available at test time within the same package. The Python and C++ generators have the same issue and that's why they have a collection of messages bundled.

Reply:
That makes sense ! I'll add it back and try to add a test build for these messages inside rosidl_generator_rs.